setup.py

```diff
@@ -481,6 +485,13 @@ def run(self):
                     no_dist=no_dist,
```

Comment:
Yeah, good point. This paramert already outdated.
But since we are currently using one func for building all this backends, I prefere to leave it here as is, but update/refactor it as follow up. I will gather all not critical suggestion for building update and create proper ticket.
I want to avoid messing up the branch with more and more refactoring changes.